src/main/java/com/redhat/quarkus/mandrel/collector/report/endpoints/ImageStatsResource.java

```diff
@@ -214,6 +230,20 @@ public ImageStats updateRunnerInfo(@PathParam("statId") Long statId, RunnerInfo
         return stat;
     }
 
+    @RolesAllowed("token_write")
+    @POST
+    @Path("runner-info")
+    public RunnerInfo createRunnerInfo(@NotNull RunnerInfo info) {
+        return collection.add(info);
+    }
+
+    @RolesAllowed("token_write")
+    @DELETE
+    @Path("runner-info/{runnerId}")
+    public RunnerInfo deleteRunnerInfo(@PathParam("runnerId") Long runnerId) {
+        return collection.deleteRunnerInfo(runnerId);
```

Comment:
Handle `null` runnerId?

Reply:
Ack.

Reply:
@jerboaa There was historical mix of NotNull, although NotNull from SmallRye as a hint, not from Hibernate Validator as a hard constraint, so I refactored that, unified, added a new test class to deal with it: ResourceErrorsTest.java